src/_cookbook/repl_derive.rs

```diff
@@ -0,0 +1,7 @@
+//! # Example: REPL (Derive API)
+//!
+//! ```rust
+#![doc = include_str!("../../examples/repl-derive.rs")]
+//! ```
+//!
+#![doc = include_str!("../../examples/repl-derive.md")]
```

Comment:
Let's drop this as we can't verify this example (and to match `repl.rs`)

Sorry, I had overlooked that this is the one example without an `md`